src/gosling-genomic-axis/axis-track.ts

```diff
@@ -17,15 +17,22 @@ const TICK_COLOR = 0x777777;
 type TickLabelInfo = {
     importance: number;
     text: PIXI.Text;
-    rope: PIXI.SimpleRope;
+    rope?: PIXI.SimpleRope | null;
 };
 
-function AxisTrack(HGC: any, ...args: any[]): any {
+function AxisTrack(HGC: typeof import('@higlass/available-for-plugins'), ...args: any[]): any {
     if (!new.target) {
         throw new Error('Uncaught TypeError: Class constructor cannot be invoked without "new"');
     }
 
-    const { absToChr, colorToHex, pixiTextToSvg, showMousePosition, svgLine } = HGC.utils;
+    const {
+        absToChr,
+        colorToHex,
+        pixiTextToSvg,
+        svgLine,
+        // @ts-expect-error did you mean `setupShowMousePosition`?
+        showMousePosition
```

Comment:
This is probably the biggest bug I found. We use this function twice but with the `setupShowMousePosition` arguments instead. Not sure what the desired behavior is.